test/examples/util/util.go

```diff
@@ -0,0 +1,247 @@
+package util
+
+//This will set the corresponding example directory
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	"gotest.tools/v3/icmd"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	//CmdTimeout defines the amount of time we should spend waiting for the resource when condition is true
+	CmdTimeout    = 10 * time.Minute
+	retryInterval = 5 * time.Second
+	retryTimeout  = 3 * time.Minute
+)
+
+var (
+	workingDirPath, _ = os.Getwd()
+	workingDirOp      = icmd.Dir(workingDirPath)
+	kubeConfig        = os.Getenv("KUBECONFIG")
+	environment       = []string{fmt.Sprintf("KUBECONFIG=%s", kubeConfig)}
+	//checkFlag         = false
+	cntr int
+)
+
+//Run runs a command with timeout
+func Run(cmd ...string) *icmd.Result {
+	currentCmd := icmd.Cmd{
+		Command: cmd,
+		Timeout: CmdTimeout,
+		Env:     environment,
+	}
+	fmt.Printf("=> Command to execute: %v \n", currentCmd)
+	if workingDirOp != nil {
+		return icmd.RunCmd(currentCmd, workingDirOp)
+	}
+	return icmd.RunCmd(currentCmd)
+}
+
+//SetDir sets the working directory
+func SetDir(path string) {
+	workingDirOp = icmd.Dir(path)
+}
+
+//SetKubeConfig sets the KUBECONFIG to the cluster
+func SetKubeConfig(kc string) {
+	kubeConfig = kc
+}
+
+//GetExamplesDir returns a root path to the examples
+func GetExamplesDir() string {
+	wd, err := os.Getwd()
+	if err != nil {
+		log.Fatalf("Failed to get the working dir.")
+	}
+	return path.Clean(fmt.Sprintf("%s/../../examples", wd))
+}
+
+/*
+//GetPodNameFromLst returns specific name of the pod from the pod list
+func GetPodNameFromLst(pods, srchItem string) (bool, string) {
+	item := ""
+	checkFlag := false
+	lstArr := strings.Split(pods, " ")
+	for _, item := range lstArr {
+		if strings.Contains(item, srchItem) {
+			if strings.Contains(srchItem, "-build") {
+				fmt.Printf("item matched as %s \n", item)
+				checkFlag = true
+				return checkFlag, item
+			}
+			return true, item
+		}
+	}
+	return checkFlag, item
+}
+*/
+
+//GetOutput returns the output using Stdout()
+func GetOutput(res *icmd.Result) string {
+	var output string
+	exitCode := res.ExitCode
+	if exitCode == 0 || exitCode == 127 {
+		output = res.Stdout()
+	} else {
+		output = res.Stderr()
+	}
+	fmt.Printf("OUTPUT: %s \n", output)
+	return output
+}
+
+//GetPjtCreationRes returns specific name of the pod from the pod list
+func GetPjtCreationRes(pjtRes string, pjt string) string {
```

Comment:
What does `pjt` stand for?

Reply:
its project. might be contradicting with k8s naming convention. I named ns as namespace in administrator mode and pjt in developer mode. It has service-binding-demo as the value. 

If required, i can change that. to nsDevMode or nsDev.

Reply:
Yes please change the name `Pjt`

Since a lot of people are contributing to the same codebase, let's use descriptive variable and function names.

Reply:
sure, will do

Reply:
Done, Renamed operatorsNS (ns) for the openshift-operators and appNS (pjt) for the service-binding-demo